### ISSUE TITLE
cockpit-[bridge|system] from your repo breaks cockpit-dashboard on Ubuntu 20.04 LTS

### ISSUE THREAD

**Issue by JOduMonT** [45Drives/cockpit-file-sharing#10]
- OS: Ubuntu 20.04.3 LTS
- Cockpit version before your repo: 

## Context
I installed cockpit from Ubuntu source and [added your repo](https://github.com/45Drives/cockpit-file-sharing#ubuntu)
to manage my Samba (which works marvellously)
but in your repo you're more than just your add-ons, you also have `cockpit-*` which under Ubuntu focal, at least break few packages

## Detail
```
Applying updates failed
The following packages have unmet dependencies:
  cockpit-system: Breaks: cockpit-dashboard but 215-1 is to be installed
  cockpit-dashboard: Depends: cockpit-ws (< 215-1.1~) but 237.2-1focal is to be installed
  cockpit-bridge: Breaks: cockpit-dashboard (< 233) but 215-1 is to be installed
```

```
apt-cache policy cockpit
cockpit:
  Installed: 215-1
  Candidate: 237.2-1focal
  Version table:
     252-1~ubuntu20.04.1 100
        100 http://archive.ubuntu.com/ubuntu focal-backports/universe amd64 Packages
     237.2-1focal 500
        500 http://images.45drives.com/repo/debian focal/main amd64 Packages
 *** 215-1 500
        500 http://archive.ubuntu.com/ubuntu focal/universe amd64 Packages
        100 /var/lib/dpkg/status
```

### 1
I was thinking of pinning your repo with a lower priority, but apparently my pinning simply won't want to work, which any will only fix the issue for myself.

```
cat /etc/apt/preferences.d/cockpit
Package: cockpit-*
Pin: origin "images.45drives.com"
Pin-Priority: 400
```

### 2
You come with a solution which could be removing those file from your repo or a better one if you know it.

### 3
You tell me

**Comment by OBXMan** [comment]
I experienced the same problem.  Not much of a techie myself, but rerunning the same command I originally used to install cockpit after having installed the plugin seemed to solve the problem.

**Comment by pweaver2019** [comment]
Wish I saw this page before I attempted to install this package. If I had a working Cockpit Manager before trying to install this what steps would get be back to a working Cockpit Manager? Right now I lost my Docker plugin, and the Samba plugin just gives me "Samba must be configured to include registry. Add `include = registry` to the [global] section of /etc/samba/smb.conf" even though I have the smb.conf setup with just "[global]" and "include = registry" and testparm from the command line works great. If I take out the include then testparm fails so I am 100% certain that I am using the registry from the command line but this plugin won't see it. I went through the trouble shooting section at https://knowledgebase.45drives.com/kb/kb450281-managing-smb-in-houston-ui/ with no luck. The smb share works fine, just this plugin won't see it. Plus the fact that I lost the docker plugin makes me want to go back.

**Comment by pweaver2019** [comment]
Luckily I was able to uninstall cockpit-file-sharing. I tried installing the original cockpit manager and docker plugin but I kept being told that the there was a required package that could not be found. So then I just restored my cockpit, apt and samba directories from this morning's backup. I had to remove the file /etc/apt/sources.list.d/45drives.sources, then reboot. Now I'm back to a working cockpit manager. Next time I'll read the issues to make sure someone hasn't already reported that a new tool will break what was working.
